src/app/shared/podcast-nav/podcast-nav-dropdown.component.css

```diff
@@ -0,0 +1,7 @@
+.dropdown-button, .open .dropdown-button {
```

Comment:
`<ui-opinion>`I do sort of wish there was more left/right padding on this button.  But that could just be me`</ui-opinion>`

Reply:
well, since I can while I'm hesitating with whether I like the shared+override CSS approach... what the heck, I can and I will. I agree it looks better with padding up there. We can figure out the menu bar padding details when we work on mobile layout.

Reply:
ugh this button padding/margin problem is a little dross, might not fix that icon button https://github.com/PRX/metrics.prx.org/issues/146 but I'm gonna keep working on it and will update in this PR.

Reply:
This icon button is a whole other issue, but I adjusted the other buttons to use padding instead of margin (I built the icon dropdown first and was reducing the styleguide btn-icon margin, still using margin with these other buttons was an oversight on my part,) and I added additional padding to the podcast-nav.

Reply:
👍